lib/features/temp/presentation/temp_flow.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_template/features/common/widgets/di_scope/di_scope.dart';
+import 'package:flutter_template/features/navigation/domain/entity/app_route_names.dart';
+import 'package:flutter_template/features/temp/di/temp_scope.dart';
+import 'package:flutter_template/features/temp/presentation/screens/temp/temp_screen.dart';
+
+/// Entry of template feature.
+@RoutePage(name: AppRouteNames.tempScreen)
+class TempFlow extends StatelessWidget implements AutoRouteWrapper {
+  /// {@macro scan_biomarkers_entry.class}
+  const TempFlow({super.key});
```

Comment:
You should not use `@macro` without previously using the `@template`. 
Add `@template` and change `@macro` for the relevant class, not `scan_biomarkers_entry.class`.